js/obstacle.js

```diff
@@ -1,152 +0,0 @@
-// Still need some comments here...
-class Obstacle {
-  constructor(x, y, w, h) {
-    this.x = x;
-    this.y = y;
-    this.w = w;
-    this.h = h;
-    this.collided = false;
-    this.r = 40;
-    this.g = 80;
-    this.b = 80;
-  }
-
-  update() {
-    let hit = collideRectCircle(this.x, this.y, this.w, this.h, player.x, player.y, player.d);
-
-    fill(this.r, this.g, this.b);
-    rect(this.x, this.y, this.w, this.h, 3);
-    // Working collision detection
-    // Split in two parts for debugging and better readability
-    let fromBelow = player.y > this.y + this.h;
-    let fromAbove = player.y < this.y;
-    let fromLeft = player.x < this.x;
-    let fromRight = player.x > this.x + this.w;
-
-    if (hit && fromBelow) player.y += 6;
```

Comment:
why is 6 the magical number here? why not 5.2 or 6.7 or 42?! ;)

Reply:
+ where else would this 6 have to change if it where to change?... nobody knows if it's not a variable we can control...